stripe-ui-core/src/test/java/com/stripe/android/uicore/utils/PaparazziConfigOption.kt

```diff
@@ -0,0 +1,33 @@
+package com.stripe.android.uicore.utils
+
+import app.cash.paparazzi.DeviceConfig
+import com.android.resources.NightMode
+
+interface PaparazziConfigOption {
```

Comment:
We should eventually add a UI testing module that contains these, so that they’re not duplicated across `identity`, `paymentsheet`, and others. Not in scope for this pull request, but we can discuss it in the next Android SDK chat.

Reply:
Yes, gradle does [support](https://stackoverflow.com/questions/5644011/multi-project-test-dependencies-with-gradle) sharing test modules. I tried to do this last year and it was not supported by android studio, it seems they [fixed](https://issuetracker.google.com/issues/139762443) it recently, we can take a look on sharing the tests